source/numerics.tex

```diff
@@ -7818,21 +7818,18 @@
 \end{codeblock}
 
 \pnum
-The
-\tcode{slice_array}
-template is a helper template used by the
+This template is a helper template used by the
 \tcode{slice}
 subscript operator
 
```

Comment:
This newline can go, too. Otherwise fine.

Reply:
(Although I wonder why need to restate the member declaration here rather than just have a "usual" reference to the corresponding section [valarray.sub].)

Reply:
We restate the member declaration all the time.